_posts/2020-11-31-homebrew-2.6.0.md

```diff
@@ -0,0 +1,36 @@
+---
+title: 2.6.0
+author: MikeMcQuaid
+---
+Today I'd like to announce Homebrew 2.6.0. The most significant changes since 2.5.0 are macOS Big Sur support, `brew` commands replacing all `brew cask` commands, the beginnings of macOS M1/Apple Silicon/ARM support and API deprecations.
```

Comment:
```suggestion
Today I'd like to announce Homebrew 2.6.0. The most significant changes since 2.5.0 are macOS Big Sur support on Intel, `brew` commands replacing all `brew cask` commands, the beginnings of macOS M1/Apple Silicon/ARM support and API deprecations.
```